# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDevices.cpp
# Commit: a0b6647eea52
# Full Hash: a0b6647eea52490282fad35f2cf3eb3d56193b26
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-10-21 21:51:55
# Description:
#   Bug 1589854 - Remove recent assert that gets triggered by latent bug. r=jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49919
# ==============================================================================

diff -r f1453e41758b -r a0b6647eea52 dom/media/MediaDevices.cpp
--- a/dom/media/MediaDevices.cpp	Mon Oct 21 12:07:10 2019 +0000
+++ b/dom/media/MediaDevices.cpp	Mon Oct 21 13:14:33 2019 +0000
@@ -165,7 +165,6 @@
           },
           [this, self, p](const RefPtr<MediaMgrError>& error) {
             nsPIDOMWindowInner* window = GetWindowIfCurrent();
-            MOZ_DIAGNOSTIC_ASSERT(!window, "Should only fail after navigation");
             if (!window) {
               return;  // Leave Promise pending after navigation by design.
             }
